### PR TITLE
Initial re-theme overrides to brackets filetree 

### PR DESCRIPTION
**Pull request by JordanTheriault**
This change removes several components from the filetree and colours it differently, resembling the spec [here](https://www.dropbox.com/s/juk9et5fmvo7hya/ui-overview.pdf?dl=0).

### CODE REVIEW

**Comment by JordanTheriault** [comment]
Below is a screenshot of the changes 
screenshot 2015-06-24 16 17 00

**Comment by sedge** [comment]
@JordanTheriault I thought you removed the drop shadow?

**Comment by JordanTheriault** [comment]
The shadow is in Thimble, not brackets.

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+	color: #666666;
+}
+
+.filetree-selection {
+	background-color: #333333;
+	border: none;
+	color: white;
+	font-weight: bolder;
+	padding: 13px;
+	width:80%;
```
Comment (humphd) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: extra space here after `:`

**Comment by humphd** [comment]
one nit, r+ with that

File: src/extensions/default/bramble/stylesheets/sidebarTheme.css (hunk 1 above)
Comment (JordanTheriault) [inline src/extensions/default/bramble/stylesheets/sidebarTheme.css]: Just noticed this is a repeat, merging with the div.filetree-selection below.

**Pull request merged**